src/registry.rs

```diff
@@ -265,4 +280,119 @@ mod tests {
             panic!("Should be a composite type definition")
         }
     }
+
+    #[test]
+    fn clear_docs() {
+        #[allow(unused)]
+        /// docs
+        struct S {
+            /// docs
```

Comment:
For tests I'd use different doc contents for every documentation line as it is more robust to testing errors.

Reply:
Also this test is actually misleading because the `TypeInfo` is handrolled and not derived.